proto/pgctld.proto

```diff
@@ -0,0 +1,211 @@
+// Copyright 2025 The Multigres Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package pgctld;
+
+option go_package = "github.com/multigres/multigres/go/pb";
+
+// PostgreSQL Control Service
+service PgCtld {
```

Comment:
These methods were not exposed via CLI (I think):  

rpc Status(StatusRequest) returns (StatusResponse);
rpc Version(VersionRequest) returns (VersionResponse);
rpc InitDataDir(InitDataDirRequest) returns (InitDataDirResponse);

Reply:
We do have status, but not the other two. Do you think they are useful? it's easy to add them.

Reply:
I've added these two.